src/v2/BaseLoginRouter.js

```diff
@@ -13,6 +13,7 @@
 // BaseLoginRouter contains the more complicated router logic - rendering/
 // transition, etc. Most router changes should happen in LoginRouter (which is
 // responsible for adding new routes)
+/* global Promise */
```

Comment:
we need that ?

Reply:
Yeah this is required to use built-in Promise. Otherwise i can import Q.

Reply:
I mean this comment.. Without that `Promise` won't be available ?